README.md

```diff
@@ -82,13 +82,27 @@ running the benchmarks from `da-control` pallet, and the generated file is
         --steps=30 \
         --repeat=20 \
         --log=warn \
-        --execution=wasm \
-        --wasm-execution=compiled \
```

Comment:
Shouldn't we use these execution strategies for benchmarking?

Reply:
They are not needed anymore :pepe_salute:

Reply:
Yep they are "used" by default now.

--execution was removed (doc is not yet updated)
--wasm-execution compiled is default